astropy/table/column.py

```diff
@@ -894,13 +883,11 @@ def __unicode__(self):
 
         lines, outs = self._formatter._pformat_col(self)
         return '\n'.join(lines)
-    if not six.PY2:
-        __str__ = __unicode__
+
+    __str__ = __unicode__
```

Comment:
Wouldn't it be better to rename `__unicode__` to `__str__`?

Reply:
Sure, though it can also be fixed later if we miss some things like this.